### DISCUSSION TITLE
Unable to open Extension:Python:Server Returned 404

### DISCUSSION THREAD

**Discussion by yuchen1204** [coder/code-server#5054] [Q&A] [answered]
![IMG_20220331_164155_HHT.jpg](https://user-images.githubusercontent.com/92127505/161014845-e4b6f27c-1eab-4378-a0d8-f9d2773f14b4.jpg)

**Comment by yuchen1204** [comment]


**Reply by jsjoeio** [reply]
Can you open the DevTools and see if there are any browser logs? Or if there are any server logs?

**Reply by yuchen1204** [reply]
I have solved the problem

**Reply by jsjoeio** [reply]
How?

**Comment by yuchen1204** [comment, marked as answer]
install from vsix
